multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -214,11 +218,20 @@ def __init__(
             If using logging, you can alternatively control amount of information printed via `logger.setLevel(L)`,
             where `L` ranges from 0 to 50
             (Note: higher values of `L` correspond to fewer print statements, opposite of verbosity levels)
+        num_classes
+            Number of classes. Used in classification task.
+            If this is specified and is different from the pretrained model's output,
+            the model's head will be changed to have <num_classes> output.
+        classes
+            All classes in this dataset.
         warn_if_exist
             Whether to raise warning if the specified path already exists.
         enable_progress_bar
             Whether to show progress bar. It will be True by default and will also be
             disabled if the environment variable os.environ["AUTOMM_DISABLE_PROGRESS_BAR"] is set.
+        init_scratch
```

Comment:
All the models in the model zoo have COCO pretrained weights. So currently the scratch option is completely from scratch. MMDet provides imagenet pretrained backbones [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/model_zoo/open_mmlab.json), and we can decide if we would like to add this after 0.6 release. This init_scratch is also for other pipelines, since currently when we init pretrained for pipeline, we set pretrained=True, without the option that only loads the structure.

Reply:
Ok. If we can't initialize from ImageNet pretrained weights, then training on COCO would take a very long time to get decent results. Let's add this after 0.6 release.